core-model/src/main/java/com/sequenceiq/cloudbreak/domain/SecurityConfig.java

```diff
@@ -144,4 +145,19 @@ public String getUserFacingKey() {
     public void setUserFacingKey(String userFacingKey) {
         this.userFacingKey = new Secret(userFacingKey);
     }
+
+    @Override
+    public String toString() {
+        return "SecurityConfig{" +
+                "id=" + id +
+                ", clientKey=" + clientKey +
```

Comment:
Do we want to log these? these are either sensitive info or vault paths, right?
clientKey
userFacingCert
userFacingKey

Reply:
Nice catch, I will remove them